### ISSUE TITLE
ReferenceError: baseUri is not defined

### ISSUE THREAD

**Issue by AwaisAhmad07** [HashLips/hashlips_art_engine#664]
I don't know why this is popping up again and again but when ever I erase this baseuri from module.export then it just export last folder of layers 
I'll be thankfull to you for your kind help

**Comment by bolshoytoster** [comment]
@AwaisAhmad07 you shouldn't remove things from the `module.exports`.

**Comment by AwaisAhmad07** [comment]
@bolshoytoster but without that running the same code as it is it shows "ReferenceError: baseUri is not defined "
it will be helpful if you can solve my issue

**Comment by bolshoytoster** [comment]
@AwaisAhmad07 just add `const baseUri = "ipfs://NewUriToReplace";` somewhere in `src/config.js`.

**Comment by AwaisAhmad07** [comment]
@bolshoytoster  thanks for your help but I'm still facing the same issue have a look at my layers
"       layersOrder: [
      { name: "Background" },
      { name: "Eyes" },
      { name: "Face" },
      { name: "watermark" },
    ],   "
 and when ever I write node index.js it only picks up the images from last layer ("watermark") and doesn't overlaps/use images from other layers/folders and when I check it in json file of built it shows metadata of other layers also have a look
  
 image": "ipfs://NewUriToReplace/1.png",
  "dna": "5c4fdfa252074a4476d0330d6d3b47444b50fa60",
  "edition": 1,
  "date": 1643916491293,
  "attributes": [
    {
      "trait_type": "Background",
      "value": "4"
    },
    {
      "trait_type": "Eyes",
      "value": "5"
    },
    {
      "trait_type": "Face",
      "value": "5"
    },
    {
      "trait_type": "watermark",
      "value": "4"
    }
  ],
  "compiler": "HashLips Art Engine"
}

"
I'll be thankfull for your kind help

**Comment by bolshoytoster** [comment]
@AwaisAhmad07 have you made sure your images are transparent and don't overlap each other?

**Comment by AwaisAhmad07** [comment]
@bolshoytoster  yes I've tried it multiple times but it only shows the some watermark image in built folder with a white background and a little bit more shiny than original and doesn't shows other layers
